src/System.IO.Pipes/tests/AnonymousPipesThrowsTests.cs

```diff
@@ -1,17 +1,15 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-using System.IO.Pipes;
-using System.Threading.Tasks;
-using Xunit;
-
 using Microsoft.Win32.SafeHandles;
+using Xunit;
 
 namespace System.IO.Pipes.Tests
 {
     public class AnonymousPipesThrowsTests
     {
+        private static void NotReachable(object obj) { Assert.True(false, "This should not be reached."); }
```

Comment:
> I assume that is the purpose of NotReachable()

Yes.  Assert.Throws would actually be the right thing here, except that as a "help" it detects that the object returned is a Task and throws an exception to say that Task-returning methods should instead use Async.ThrowsAsync.